test/services/network/translator/test_network_translator.py

```diff
@@ -223,23 +223,12 @@
     ###############################
 
     "create_circuit": create_circuit(),
-}
 
+    "create_location": create_location(),
```

Comment:
todo: This should be moved to the correct location in the map (ok, ok, dict) based on the object hierarchy.